src/app/screens/ConfirmSignMessage/index.tsx

```diff
@@ -55,58 +54,65 @@ function ConfirmSignMessage(props: Props) {
   }
 
   return (
-    <div>
-      <PublisherCard
-        title={originRef.current.name}
-        image={originRef.current.icon}
-      />
-      <div className="py-4">
-        <Container maxWidth="sm">
-          {!succesMessage ? (
-            <>
-              <dl className="shadow bg-white dark:bg-surface-02dp p-4 rounded-lg mb-8">
-                <dt className="font-semibold text-gray-500">
-                  {originRef.current.host} asks you to sign:
-                </dt>
-                <dd className="mb-6 dark:text-white">{messageRef.current}</dd>
-              </dl>
-
+    <div className="h-full flex flex-col overflow-y-auto no-scrollbar">
+      <div className="text-center text-xl font-semibold dark:text-white py-2 border-b border-gray-200 dark:border-neutral-500">
+        Sign
+      </div>
+      {!successMessage ? (
+        <div className="h-full flex flex-col justify-between">
+          <div>
+            <PublisherCard
+              title={originRef.current.name}
+              image={originRef.current.icon}
+            />
+            <dl className="m-4 shadow bg-white dark:bg-surface-02dp p-4 rounded-lg">
```

Comment:
Kinda looks like we can also move this into it's own component to keep all views consistent.